ebpf/tracer.go

```diff
@@ -136,16 +216,42 @@ func (t *Tracer) getConnections() ([]ConnectionStats, error) {
 	// Remove expired entries
 	t.removeEntries(mp, tcpMp, expired)
 
-	return active, nil
+	// Add the closed connections
+	t.closedConnsLock.Lock()
+	for i := 0; i < len(t.closedConns); i++ {
+		t.closedConns[i].collected = true
+	}
+	active = append(active, t.closedConns...)
+	t.closedConnsLock.Unlock()
+
+	return removeDuplicates(active), nil
+}
+
+func (t *Tracer) cleanupClosedConns() {
```

Comment:
This is confusing to me - why aren't we just setting `t.closedConns = []ConnectionStats{}` above after we've included them all into active? Is it so that multiple calls to `getActiveConnections()` will include this data?

Reply:
Yes this is related to: https://github.com/DataDog/datadog-process-agent/pull/233#discussion_r243665986

Reply:
Hmm - I'm not sure this makes sense to me; If we're running this on a 10s timer, isn't there a possibility that there's an inconsistency between `getActiveConnections()` calls which happen at the 10s interval in another binary (which may not align with this one)

Reply:
This is actually an issue that affects the `BytesSinceLastCheck` feature I was working on - keeping this state in process-agent causes correctness issues when restarting the agent, and keeping it in the network-tracer causes correctness issues on multiple curls.